pkg/helm/chart.go

```diff
@@ -41,9 +41,21 @@ func FetchChaosMeshChart(ctx context.Context, version string) (*chart.Chart, err
 	return requestedChart, nil
 }
 
+func GetChaosMeshChartTgzPath(ctx context.Context, version, local string) (string, error) {
+	fileName := fmt.Sprintf("/chaos-mesh-%s.tgz", version)
+	tgzPath := fmt.Sprintf("%s/%s", os.TempDir(), fileName)
+	if local != "" {
+		tgzPath = fmt.Sprintf("%s/%s", local, fileName)
+	}
+
+	if _, err := os.Stat(tgzPath); err != nil {
+		return DownloadChaosMeshChartTgz(ctx, version)
+	}
```

Comment:
please do not fallback to the `DownloadChaosMeshChartTgz`, just return the error.

the fallback behavior would make it complex and ambiguous.